rebuild-docs

```diff
@@ -12,6 +12,13 @@ rm -rf ${core_dir} && git clone https://github.com/canonicalltd/ubuntu-core-docs
 rm -rf ${phone_dir} && git clone https://github.com/ubuntudesign/phone-docs.git ${phone_dir} || true
 #git clone https://github.com/canonicalltd/snapcraft-docs.git ${snapcraft_dir}
 
+# Extra build steps for Ubuntu Core docs
+(
+  cd ${core_dir}
+  repo init -u git@github.com:CanonicalLtd/ubuntu-core-docs.git
```

Comment:
I will be snapping up git-repo soon and making it available in the public store. Expect this in the next couple of weeks.